README.md

```diff
@@ -60,11 +60,13 @@ func main() {
 Create a VPS
 
 ```go
+enableIP6 := false
+
 instanceOptions := &govultr.InstanceCreateReq{
   Label:                "awesome-go-app",
   Hostname:             "awesome-go.com",
-  Backups:              true,
-  EnableIPv6:           true,
+  Backups:              "false",
+  EnableIPv6:           &enableIP6,
```

Comment:
What you wrote here is good! But to make your life easier GoVultr has a helper function `BoolToBoolPtr` which will cut back on some of the code.

`govultr.BoolToBoolPtr(false)`

Reply:
Thanks for the tip, definitely still getting my head back into working with points after a long hiatus.